client/src/pages/OfferHelp.js

```diff
@@ -102,12 +109,21 @@ const Step2 = (props) => {
         <div style={{ marginBottom: "40px", textAlign: "center" }}>
           <LocationInput
             location={props.location}
+            gtmPrefix={GTM.requestHelp.prefix + props.currentStep}
```

Comment:
I think this is the wrong prefix? You'll need to add a `prefix: OFS` under the `offerHelp` property.

Reply:
For components that are reused in different contexts like `LocationInput` and `WizardNav`, we simply have to pass in the prefix (which is OFS in this case). The buttons that need to be tracked in those components already have the end of the tag set. For example, see [here](https://github.com/FightPandemics/FightPandemics/blob/bb660d0853b5ecdce501c25c8b91b8249cebaab3/client/src/components/Input/LocationInput.js#L159). You just have to pass in a prefix to `LocationInput`.